test/benchmark/promises/shared.js

```diff
@@ -25,7 +25,6 @@ const tests = [
       const promises = [prom]
 
       for (let i = 0; i < internalPromises; ++i) {
-        prom = promises[i]
```

Comment:
I think you still want this part as it is supposed to be chaining promises. My point was we don't need to reuse the outer 'prom'. The outer prom is just used to seed 'promises' and the inner could be scoped const just to the for loop. Its all not very critical but I think I unfortunately led you to changing what is being tested here. This setup is sort of awkward.

Reply:
lol, i see it now. my bad